requirements.txt

```diff
@@ -0,0 +1,20 @@
+asgiref==3.2.10
```

Comment:
why do we have requirements file in react app?

Reply:
Initially I created it because I thought its required to list packages installed using npm but it didn't include those packages. Forgot to remove it later on.